app/login/page.tsx

```diff
@@ -1,8 +1,17 @@
+import Image from "next/image";
+
 import LoginForm from "./_components/LoginForm";
 
```

Comment:
`layout.tsx`의 `<meta name="theme-color"... />` 부분을 제거하고 아래처럼 수정하면 상단바까지 색상을 적용할 수 있어요. 🙂
 [참고 링크](https://nextjs.org/docs/app/api-reference/functions/generate-viewport) 

```suggestion

export const viewport: Viewport = {
  themeColor: "#E8E8E8",
};

```

<img src="https://github.com/Open-Eye-Im-Developer/MoGakGo-FE/assets/86952779/112cb2f7-a835-4288-9ea6-9e106222a910" alt="image" width="250px" />

Reply:
오 와우 감사합니다!!

Reply:
현재 main, login, reivew 페이지 별로 다 색상이 다른 걸로 알아서 이 부분은 다른 브랜치에서 같이 적용해볼게요!